deploy/testenv/env/integration/backends/sqlite.yaml

```diff
@@ -6,7 +6,7 @@ cluster:
     uri: null
   compute:
     uri: null
-data_backend: sqlite:///deploy/testenv/cache/test_sql.db
+data_backend: sqlite://
```

Comment:
Don't we need to specify the location of the database file?

Reply:
sqlite:// means it stays in memory 
hence we don't need to clean again and again on each test.